management_api_app/db/events.py

```diff
@@ -10,14 +11,56 @@ async def connect_to_db(app: FastAPI) -> None:
     logging.debug(f"Connecting to {config.STATE_STORE_ENDPOINT}")
 
     try:
-        cosmos_client = CosmosClient(config.STATE_STORE_ENDPOINT, config.STATE_STORE_KEY)
+        cosmos_client = CosmosClient(config.STATE_STORE_ENDPOINT, config.STATE_STORE_KEY, connection_verify=False)
         app.state.cosmos_client = cosmos_client
         logging.debug("Connection established")
     except Exception as e:
         logging.debug(f"Connection to state store could not be established: {e}")
 
+# Bootstrapping is temporary while the API does not have a register spec api implemented.
+
+
+async def create_bundle_registry(database: DatabaseProxy):
+    """
+    Creates a bundle spec container if one does not exist and populate it with a canonical spec.
+    :param database: DatabaseProxy for STATE_STORE_DATABASE
+    :returns: None
+    """
+    bundle_spec = {
```

Comment:
Discussing with @Lybecker @jjcollinge @askew and they brought up that we want to decouple the API from the backend tech we use - so perhaps resource_spec and /ResourceSpecs would be an name that allows us to decouple, and then re-thinking the url - we could maybe make the URL the id - or similar

Reply:
resource spec is a good idea. Best to deep the domain model separate. id I am not so sure about. Think about explaining id to someone. Well the id is the url where the resource spec exists and then so why not use url? It's like name for docker container one specifies the full docker path to it including the registry name so it is a full url